prac_09/cleanup_files.py

```diff
@@ -43,12 +42,29 @@ def main():
 def get_fixed_filename(filename):
     """Return a 'fixed' version of filename."""
     # First, replace the spaces and .TXT (the easy part)
-    filename = filename.replace(" ", "_").replace(".TXT", ".txt")
+    new_name = filename.replace(" ", "_").replace(".TXT", ".txt")
+    new_name = new_name[0].upper() + new_name[1:]
 
-    new_name = ""
-    # TODO: step-by-step, consider the problem cases and solve them
+    # Second, replace PascalCase. i.e. SilentNight
+    pascal_name = new_name[0]
```

Comment:
These different names are misleading. I would suggest you just use one "new_name" variable. You don't need to keep different versions - only the final, new name.